charts/argo-cd-ha/values.yaml

```diff
@@ -0,0 +1,53 @@
+argo-cd:
+  installCRDs: false
+  controller:
+    extraArgs:
+      sentinel=argocd-redis-ha-announce-0:
```

Comment:
Maybe we should handle the key/value mapping differently if the redis-ha chart is added to the argo-cd chart... Something like if redis-ha.enabled -- We could be more clear about adding these args into the deployments.

Reply:
The "problem" with that is to find the balance between how much you want to hide behind that flag and how much flexibility you give to the user to deploy the chart...
But yes, it's a good idea. I'll try some GoLang templates (aka `_helpers.tpl`) to see if I can get that flexibility and default values

Reply:
I raised a PR to improve this weird syntax

https://github.com/argoproj/argo-helm/pull/244